main/src/com/google/refine/exporters/sql/SqlCreateBuilder.java

```diff
@@ -83,11 +84,12 @@ public String getCreateSQL() {
                 
                 if (name != null) {
                     if(trimColNames) {
-                        String trimmedCol = name.replaceAll("[^a-zA-Z0-9_]", "_");
-                        createSB.append( trimmedCol + " ");
-                    }else{
-                        createSB.append(name + " ");
+                        name = name.replaceAll("[^a-zA-Z0-9_]", "_");
+
+                    }if (EscapeCharacter!=null){
+                        name=EscapeCharacter+name+EscapeCharacter;
                     }
```

Comment:
@the-star-sea: here, your name for the variable starts with an uppercase. But, other options are in lowercase. Also, the name could be more precise: since we have `trimColNames`, `escapeColNames` is close, yet significative and different enough. `EscapeCharacter` doesn't really say to what it applies to, only what it does. `escapeColNamesCharacter` say both.